Valet.xcodeproj/project.pbxproj

```diff
@@ -1117,9 +1117,9 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 16C3B072204B1DB800B4D0B4 /* Build configuration list for PBXNativeTarget "Valet tvOS" */;
 			buildPhases = (
+				16C3B06A204B1DB800B4D0B4 /* Headers */,
```

Comment:
These changes fixes an occasional "build cycle detected" error. Headers should always be compiled before Sources to avoid this.